### ISSUE TITLE
Instagram does not receive messages

### ISSUE THREAD

**Issue by marcelocaciatori** [chatwoot/chatwoot#8162]
### Describe the bug

Follow the entire chatwoot documentation process in a video

https://www.awesomescreenshot.com/video/21794913?key=74443d1e93913a3e6540db711163368b

Below is a print of the data placed in Chatwoot  .env

![36-61-523900-a0](https://github.com/chatwoot/chatwoot/assets/91642837/f63ad6a5-0d5e-423d-a1e7-2310398e9e70)

### To Reproduce

We configure the app in Meta Develop and try to receive and send messages

### Environment

Linux VM

### Cloud Provider

None

### Platform

Browser

### Operating system

Ubuntu 22.04

### Additional context

Facebook works perfectly, but Instagram doesn't send or receive messages

We remind you that all necessary resources are approved in the Facebook account

chatwoot logs nothing related to the case

**Comment by pranavrajs** [comment]
@marcelocaciatori I don't see the Instagram configured as per the docs here. https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/instagram-channel-setup. 

Please set it up. If it is still an issue, feel free to reopen the issue.

**Comment by shivamkb17** [comment]
This issue is rising in recent time, earlier this is type of issue is not showing, A strange issue is showing. 
I created a facebook bussiness inbox, with connected instagram bussiness id. 

I am getting response from the some user, and from some user, no any message is coming on the instagram message in chatwoot inbox.

**Comment by shivamkb17** [comment]
I am debugging It, I am also planning this here to submit about this issue. First I thought this issue submit here, and I found a bug report on facebook developer community forum. 

https://developers.facebook.com/support/bugs/1941620312886486/

There are some issue facing in instagram graph API. 

I tested it by installing chatwoot several time, even in digital ocean server and in google cloud, I face similar issue always.
